packages/create-next-app/index.ts

```diff
@@ -43,6 +43,22 @@ const program = new Commander.Command(packageJson.name)
   .parse(process.argv)
 
 async function run(): Promise<void> {
+  // NodeJs Version Validation
+  var currentNodeVersion = process.versions.node
```

Comment:
let nodeVersion = parseFloat(process.versions.node);
if (nodeVersion < 10.13) {
  throw new Error(`Your current node version ${nodeVersion} is not supported! The minimum supported version is 10.13 and above!`)
}